handel/src/services/route53zone/index.ts

```diff
@@ -75,21 +75,21 @@ function getCompiledRoute53Template(ownServiceContext: ServiceContext<Route53Zon
  */
 
 export function check(serviceContext: ServiceContext<Route53ZoneServiceConfig>, dependenciesServiceContexts: Array<ServiceContext<ServiceConfig>>): string[] {
-    const errors = [];
+    const errors: string[] = checkPhase.checkJsonSchema(`${__dirname}/params-schema.json`, serviceContext);
 
     const params = serviceContext.params;
 
     if (!params.name) {
-        errors.push(`${SERVICE_NAME} - 'name' parameter must be specified`);
+        errors.push(`'name' parameter must be specified`);
     } else if (!route53.isValidHostname(params.name)) {
-        errors.push(`${SERVICE_NAME} - 'name' parameter must be a valid hostname`);
+        errors.push(`'name' parameter must be a valid hostname`);
     }
 
     if (params.private && typeof params.private !== 'boolean') {
```

Comment:
This if statement should also be able to be removed because it's checked in the JSON schema now.